input/docs/index.md

```diff
@@ -1 +1,7 @@
-Welcome to the Documentation!
\ No newline at end of file
+<div class="container">
```

Comment:
This is a markdown file, please use markdown syntax.

Reply:
This was already there. I was just adding to it but I can replace this with markdown. Looks like the links and images should be in HTML though, as that accomadates buttons, sizing etc.